### DISCUSSION TITLE
Welcome to acnportal Discussions!

### DISCUSSION THREAD

**Discussion by zach401** [zach401/acnportal#95] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share how you are using acnportal.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AimanKhan1997** [comment]
Mohd Aiman Khan, Master student at KTH Sweden, working on autonomous shared DCFCs in ACN-SIM

**Comment by sunash** [comment]
Hi everyone! I'm Sunash Sharma, a software engineer and researcher at Caltech. Aside from contributing to and maintaining acnportal, I also use acnportal as the base in OpenAI Gym environments for adaptive EV charging, implemented here: https://github.com/caltech-netlab/gym-acnportal Please reach out to @zach401 or me if you have any questions!
